### DISCUSSION TITLE
Composite, S-VHS, BetaMax, U-Matic

### DISCUSSION THREAD

**Discussion by Unit20** [ntsc-rs/ntsc-rs#161] [Presets]
This took a few weeks to create all of these presets. U-Matic is still being worked on but I've decided to share eitherway.
Feedback is appreciated.

 [BetaMax.json](https://github.com/user-attachments/files/16576355/BetaMax.json)
[Composite.json](https://github.com/user-attachments/files/16576356/Composite.json)
[S-VHSEffectSP.json](https://github.com/user-attachments/files/16576357/S-VHSEffectSP.json)
[U-Matic(WIP).json](https://github.com/user-attachments/files/16576358/U-Matic.WIP.json)
![Comparison_000000](https://github.com/user-attachments/assets/4d670a54-90d2-450a-9c41-36d6d08ef758)

**Comment by fuki98** [comment]
AWESOME!

**Comment by TheGithubberpng** [comment]
we need a betacam sp effect
